gems/pending/openstack/events/openstack_rabbit_event_monitor.rb

```diff
@@ -42,7 +42,7 @@ def self.test_connection(options = {})
       $log.info("#{log_prefix} The Openstack AMQP service may be using a different provider."\
                 " Enable debug logging to see connection exception.") if $log
       $log.debug("#{log_prefix} Exception: #{e}") if $log
-      return false
+      raise e
```

Comment:
I wonder if you should raise a wrapped exception so that you don't have to add bunny logic into the manageiq/providers/openstack/manager_mixin.rb .  That way it keeps the bunny specifics encapsulated down deep so the caller don't have to know or care.